docusaurus/docs/get-started/get-started.mdx

```diff
@@ -15,6 +15,28 @@ import ScaffoldNPM from '@snippets/createplugin-scaffold.npm.md';
 import ScaffoldPNPM from '@snippets/createplugin-scaffold.pnpm.md';
 import ScaffoldYarn from '@snippets/createplugin-scaffold.yarn.md';
 
+# Grafana plugin developer's guide
```

Comment:
I think if we add this header, we need more of an index of links / journey. 

The hypothesis (this could be wrong) is that we want getting started (generally) to be a "quick start" so a developer can get something up and running, and then explore the working code and the rest of this site to gain deeper knowledge as and when appropriate. 

If we remove this header, I think we can still provide a basic introduction to improve the getting started experience and partly address the feedback that led to this PR.

Reply:
I think what we probably need is a hybrid of an Intro and Get started with an emphasis on the Get started. So, 've changed the top header to "Get started with plugins" and then streamlined the following text so that there is a bit of an intro but not so much that it gets in the way of the user starting their journey.